shell/browser/api/atom_api_app.cc

```diff
@@ -471,7 +471,7 @@ void OnClientCertificateSelected(
 
   mate::Dictionary cert_data;
   if (!mate::ConvertFromV8(isolate, val, &cert_data)) {
-    args->ThrowError("Must pass valid certificate object.");
+    args->ThrowError("Valid certificate object parameter required.");
```

Comment:
For what? Can we tell them the method they called, or thing they did to trigger this error. E.g.

```suggestion
    args->ThrowError("Valid certificate object parameter required in the 'select-client-certificate' event callback.");
```